lib/plugins/aws/package/compile/events/apiGateway/lib/method/index.test.js

```diff
@@ -603,6 +603,35 @@ describe('#compileMethods()', () => {
     });
   });
 
+  it('should set required to true when omitted from mapped value', () => {
+    awsCompileApigEvents.validated.events = [
```

Comment:
All new tests should be introduced with `runServerless` util, see https://github.com/serverless/serverless/tree/master/test#unit-tests

Relying on mocked  `awsCompileApigEvents` and other classes like that, is very painful for eventual future refactors